src/main/java/io/cdap/plugin/gcp/gcs/Formats.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.gcp.gcs;
+
+import io.cdap.plugin.format.FileFormat;
+
+/**
+ * Utility class for GCS formats.
+ */
+public class Formats {
+
+  private Formats() {
+    // no-op
+  }
+
+  /**
+   * Get the plugin name for the given format. Older versions of the source and sink allowed case insensitive
```

Comment:
If it returns plugin name, should it return "format-avro"? It's the artifact name I believe

Reply:
we want the plugin name, which is what gets passed to CDAP's usePlugin() API call at https://github.com/cdapio/hydrator-plugins/blob/19a6d06a538714a5fdcde15eac9084b5bfe2fbba/format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSource.java#L124